SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-cases.adoc

```diff
@@ -5,28 +5,94 @@ NOTE:  This file integrates the use cases that are linked to the SDPi profiles t
 {empty} +
 Note that each use case is in its own file. +
 {empty} +
-Note that AsciiDoc Gherkin (plugin?) may be used
+Note that AsciiDoc Gherkin (plugin?) may be used ... eventually ... !
 
 [appendix#vol1_appendix_c_dpi_use_cases,sdpi_offset=C]
-== Appendix C - DPI Use Cases
+== Appendix C - Device Point-of-care Interoperability (DPI) Use Cases
 
 // C.1
 [sdpi_offset=1]
-=== Overview of DPI Narratives, Use Cases & Scenarios
+=== General Overview of DPI Use Cases & Analysis
 
 // C.1.1
-==== Rich History of Medical Device Interoperability Use Cases
+==== Rich History of Medical Device Interoperability (MDI) Use Cases
+
+The vision of plug-and-play medical device interoperability has been an active pursuit since the early 1980's, with the IEEE 1073 group's formation in those early years.  Over the 40+ years, many projects -- both industry and standards-based -- have contributed to an ever growing set of real-world use cases, and the ISO/IEEE 11073 SDC program is no different.
+
+Looking to leverage this wealth of use cases in considering SDPi, a "compendium of medical device oriented use cases" was created to facilitate referencing and use.  The use cases detailed in this appendix build on those that are captured in this document:  <<ihe_pcd_sdpi_use_cases_compendium_2019>>
 
 // C.1.2
 ==== Overview of Architectural & Business Systems Concepts
 
+#TODO:  Include difference between logical actors and business (real world implementation) systems#
+
 // C.1.3
-==== Overview of DIS & DAS Medical Device Alert Distribution Systems
+==== Application of ISO/IEC 60601-1-8 Concepts & Definitions
+
+include::tf1-ch-c-60601-1-8.adoc[]
+
+==== Use Cases, MBSE Requirements Modeling & SysML 2.0
+
+#TODO: MOVE THIS UP TO APPENDIX A#
+
+#TODO:
+- MBSE Reference incl. Requirements Modeling
+- Reference SysML 2.0 7.20 + intro slides on <topics>
+- ALL USE CASE FEATURE are REQUIREMENTS
+- MBSE Requirements Modeling #
+
+OMB's Systems Modeling Language 2.0 (see <<omg_sysml_2-0_spec>> and <<omg_sysml_2-0_intro_graphical_model>>), provides extended support for requirements modeling that not only provides the foundation for MBSE's Requirements Modeling, but also a computable specification that enables automated verification (e.g., using "Verification Cases").
```

Comment:
```suggestion
OMG's Systems Modeling Language 2.0 (see <<omg_sysml_2-0_spec>> and <<omg_sysml_2-0_intro_graphical_model>>), provides extended support for requirements modeling that not only provides the foundation for MBSE's Requirements Modeling, but also a computable specification that enables automated verification (e.g., using "Verification Cases").
```

Reply:
Those terms should be defined in the glossary and referenced her accordingly.

Reply:
OMG and SDO added to end of Glossary file BUT it isn't clear that this is where they should go given the nature of the file.  This is a question for Mary Jungers, IHE Publication specialist, when we are preparing the SDPi 1.0 comment version.